modules/custom/az_dam/config/quickstart/entity_browser.browser.ib_dam.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 86e23d29-bcfd-4d90-ac08-d9ba6c8d6957
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.